cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -86,14 +86,14 @@ def _finalize():
 @pytest.mark.meta(blockers=[BZ(1544535, forced_streams=['5.9'])])
 @pytest.mark.parametrize('context', [ViaSSUI])
 @pytest.mark.parametrize('order_service', [['console_test']], indirect=True)
-@pytest.mark.uncollectif(lambda provider:
-    provider.one_of(VMwareProvider) and provider.version >= 6.5 or
-    'html5_console' in provider.data.get('excluded_test_flags', []),
-    'VNC consoles are unsupported on VMware ESXi 6.5 and later')
 def test_vm_console(request, appliance, setup_provider, context, configure_websocket,
         configure_console_vnc, order_service, take_screenshot,
         console_template, provider):
     """Test Myservice VM Console in SSUI."""
+    if (provider.one_of(VMwareProvider) and provider.version >= 6.5 or
+            'html5_console' in provider.data.get('excluded_test_flags', [])):
+        pytest.skip('VNC consoles are unsupported on VMware ESXi 6.5 and later')
```

Comment:
:+1: 
But I would prefer just adding `test_flag` and excluded test_flag for provider

Reply:
I just move logic from uncollectif into test. every change requires a lot of testing. so, any optimization on this step is frowned upon